src/applications/third-party-app-directory/sass/index.scss

```diff
@@ -31,4 +38,8 @@
       margin-left: 86px;
     }
   }
+
+  button.va-button-link {
+    display: block;
```

Comment:
might be able to use the `display` [utilities](https://design.va.gov/utilities/display)